financial-connections/src/main/java/com/stripe/android/financialconnections/features/partnerauth/SharedPartnerAuthState.kt

```diff
@@ -6,21 +6,22 @@ import com.airbnb.mvrx.Loading
 import com.airbnb.mvrx.MavericksState
 import com.airbnb.mvrx.PersistState
 import com.airbnb.mvrx.Success
-import com.airbnb.mvrx.Uninitialized
 import com.stripe.android.financialconnections.model.DataAccessNotice
 import com.stripe.android.financialconnections.model.FinancialConnectionsAuthorizationSession
 import com.stripe.android.financialconnections.model.FinancialConnectionsInstitution
+import com.stripe.android.financialconnections.model.FinancialConnectionsSessionManifest
 
-internal data class PartnerAuthState(
+internal data class SharedPartnerAuthState(
     /**
      * The active auth session id. Used across process kills to prevent re-creating the session
      * if one is already active.
      */
     @PersistState
-    val activeAuthSession: String? = null,
```

Comment:
can you keep the default values so no values need to be passed when initizlied?

Reply:
actually yes, except for the pane (that is variable across screens) - https://github.com/stripe/stripe-android/pull/7394/commits/8f8d73701cfc85fd74e90349ae7072eba6cf0e05